kafka/pykafka/partition.py

```diff
@@ -36,20 +35,48 @@ def isr(self):
     def topic(self):
         return self._topic
 
-    def fetch_offsets(self, offsets_before, max_offsets=1):
+    def fetch_offset(self, consumer_group):
```

Comment:
I used `fetch_offset_limit` and `latest/earliest_available_offset` to differentiate the names in `Partition`.